__tests__/get-options-overrides.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { afterAll, test, expect } from "@jest/globals";
+import * as path from "path";
+import * as ts from "typescript";
+import { remove } from "fs-extra";
+
+import { makeStubbedContext } from "./fixtures/context";
+import { setTypescriptModule } from "../src/tsproxy";
+import { IOptions } from "../src/ioptions";
+import { getOptionsOverrides, createFilter } from "../src/get-options-overrides";
+
+setTypescriptModule(ts);
+
+const local = (x: string) => path.resolve(__dirname, x);
+const cacheDir = local("__temp/get-options-overrides");
+
+afterAll(() => remove(cacheDir));
+
+const defaultConfig: IOptions = {
+	include: ["*.ts+(|x)", "**/*.ts+(|x)"],
+	exclude: ["*.d.ts", "**/*.d.ts"],
+	check: false,
+	verbosity: 5,
+	clean: false,
+	cacheRoot: cacheDir,
+	cwd: local(""),
+	abortOnError: false,
+	rollupCommonJSResolveHack: false,
+	typescript: ts,
+	objectHashIgnoreUnknownHack: false,
+	tsconfigOverride: null,
+	useTsconfigDeclarationDir: false,
+	tsconfigDefaults: null,
+	sourceMapCallback: (id: string, map: string): void => {
+		console.log(id + map);
+	},
+	transformers: [(ls: ts.LanguageService) => {
+		console.log(ls);
+		return {};
+	}],
+};
+
+const forcedOptions: ts.CompilerOptions = {
+	allowNonTsExtensions: true,
+	importHelpers: true,
+	inlineSourceMap: false,
+	moduleResolution: ts.ModuleResolutionKind.NodeJs,
+	noEmit: false,
+	noEmitHelpers: false,
+	noResolve: false,
+	outDir: local(`${cacheDir}/placeholder`),
```

Comment:
Oh of course I found another Windows right before merging...

```
    -   "outDir": "D:\\a\\rollup-plugin-typescript2\\rollup-plugin-typescript2\\__tests__\\__temp\\get-options-overrides\\placeholder",
    +   "outDir": "D:\\a\\rollup-plugin-typescript2\\rollup-plugin-typescript2\\__tests__\\__temp\\get-options-overrides/placeholder",
```

This is because `get-options-overrides.ts` doesn't do a normalization when interpolating `/placeholder`, so this is an actual bug in _this_ codebase.

As this PR is just tests (no source changes) and about to be merged, just going to leave a `// TODO: ` comment here to fix this in the source in a future PR.
That may or may not be breaking? I'm surprised that some of this code still runs fine on Windows despite bugs like these... I'm guessing there might be some more redundant normalization done elsewhere which is why it doesn't throw an error?

Reply:
Fixed this in #331